test_ros2cli/test/test_process_output_customizable.py.in

```diff
@@ -40,11 +41,16 @@ def generate_test_description(config, ready_fn):
         output='screen',
         sigterm_timeout=LaunchConfiguration('sigterm_timeout', default=60)
     )
+    if '@RMW_IMPLEMENTATION@' == 'rmw_connext_cpp':
```

Comment:
Since the test code is otherwise RMW agnostic, I would rather see this logic moved to the `tests` macro in CMakeLists.txt. Also, with a comment explaining why this is required.